samples/client/others/rust/hyper/composed-oneof/src/models/obj_a.rs

```diff
@@ -1,3 +1,5 @@
+use crate::models;
```

Comment:
It's now aligned with server-side generator via `AbstractRustCodegen`. So a lot of sample files have only this change